test/Minitwit.CoreTests/Entities/AuthorDTOTests.cs

```diff
@@ -66,7 +66,7 @@ public void AuthorDTO_IndexAttribute_ShouldBeUnique()
         Assert.True(indexAttribute.IsUnique);
     }
 
```

Comment:
At this point I think you can just delete it.

Reply:
Pffft.. Who cares